CHANGELOG.txt

```diff
@@ -5,6 +5,10 @@ master:
  - obspy.io.shapefile:
    * Add possibility to add custom database columns when writing catalog
      objects to shapefile (see #2012)
+ - obspy.signal:
+   * PPSD: for safety reasons, raise an ObsPyException if trying to read a PPSD
+     npz file that was written with a newer ObsPy version than is currently
```

Comment:
That's not really correct. Best rephrase to "written with a newer internal file format" or something like it. I guess the format does not change that often.

Reply:
Yeah, you're right, that's misleading..